### DISCUSSION TITLE
`ttkwidgets` and `ttkthemes` Integration

### DISCUSSION THREAD

**Discussion by sumeshir26** [tukaan/tukaan#85] [Ideas]
Would this be possible since you are a member of TkinterEP?

**Comment by rdbende** [comment]
I don't think so.
Both `ttkthemes` and `ttkwidgets` are built for Tkinter, and heavily depend on it. They use Tkinter objects and often internal functions of Tkinter, that would be hard to even emulate in Tukaan.
I really think a `ttkthemes` integration would be useful though.
